### PR TITLE
CTECH-2450 use local data sets for TraderMade example

### PR DESCRIPTION
**Pull request by rizwansaeed**

### CODE REVIEW

File: examples/features/data-feeds/Using TraderMade FX spot price data in LUSID.ipynb [hunk 1]
```diff
@@ -2,7 +2,7 @@
  "cells": [
```
Comment (adamMQM) [inline examples/features/data-feeds/Using TraderMade FX spot price data in LUSID.ipynb:2]: <p>your*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/605/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (adamMQM) [inline examples/features/data-feeds/Using TraderMade FX spot price data in LUSID.ipynb:2]: <p>Would this be better in a function? something like</p><p> </p><pre class="ql-syntax" spellcheck="false">def get_quotes(tradermade_enabled = False):
if tradermade_enabled:
  quotes_response = quotes_api.list_quotes(scope=scope, filter = f"QuoteId.EffectiveAt gte 2020-03-19 and QuoteId.EffectiveAt lte {end_date}" )
  return lusid_response_to_data_frame(quotes_response, column_name_mapping=json_mappings)
else: return pd.read_csv("data/fx_rates.txt", sep='\t')
</pre><p> </p><p>that way users can just set or unset the flag - could even just set a global variable at the start of the notebook actually.</p><p>		</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/605/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (adamMQM) [inline examples/features/data-feeds/Using TraderMade FX spot price data in LUSID.ipynb:2]: <p>same here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/605/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (rizwansaeed) [reply examples/features/data-feeds/Using TraderMade FX spot price data in LUSID.ipynb:2]: good shout

**Review by adamMQM — APPROVED** [review]

**Pull request merged**